management/src/main/scala/akka/management/scaladsl/HealthChecks.scala

```diff
@@ -33,14 +33,14 @@ abstract class HealthChecks {
   /**
    * Returns Future(true) if the system is ready to receive user traffic
    */
-  def ready(): Future[Boolean]
+  def ready(): Future[Either[String, Unit]]
```

Comment:
This changes the public API in a non-compatible way. 

We'd have to add new methods and delegate

Reply:
Okay, that should be simple enough, I'll update when I have some time.